### PR TITLE
New implementation of firmware selector

### PR DESCRIPTION
**Pull request by PaulWieland**
Two step selection process. First for protocol, second for ratgdo hw version.

image

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -312,19 +277,38 @@ <h3>Advanced Users</h3>
...
+      }
+
+      if(protocol === "secplusv2"){
+        protocol = "";
+      }else{
+        protocol = `_${protocol}`;
+      }
+
+      button.manifest = `${hardware}${protocol}-manifest.json`;
+      console.log(button.manifest);
```
Comment (bdraco) [inline static/index.html]: Left over console log

**Comment by bdraco** [comment]
Are there 2.5 esp32s?

**Comment by PaulWieland** [comment]
> Are there 2.5 esp32s?

Unknown. There are a tiny percentage of v2 shields and an equally tiny percentage of v2.5 shields, none of which were sold with an esp32.

File: static/index.html (hunk 1 above)
Comment (PaulWieland) [reply static/index.html]: Yeah, made it easier to verify that the logic was producing the right manifest src. Will remove.

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Oops, looks like we are missing the image for the v2.5 esp32

**Pull request merged**